Makefile

```
@@ -30,21 +41,37 @@
-	pre-commit run --all-files
-
-format: ## Format code with Black and isort
-	@echo "✨ Formatting code..."
+lint: ## Run all linting and formatting checks (backend + frontend)
+	@echo "🔍 Running all linting checks..."
+	@echo "🐍 Running backend linting..."
+	$(MAKE) lint-backend
+	@echo "⚛️  Running frontend linting..."
+	$(MAKE) lint-frontend
+
+format: ## Format all code (backend + frontend)
```

checkmake (0.2.2)

[warning] Target body for "format" exceeds allowed length of 5 (6).

(maxbodylength)